Apps/AdminWebClient/src/Server/Services/DashboardService.cs

```diff
@@ -95,15 +97,16 @@ public int GetRecurrentUserCount(int dayCount, string startPeriod, string endPer
         }
 
         /// <inheritdoc />
-        public RequestResult<IEnumerable<Database.Models.MessagingVerification>> GetMessageVerifications(UserQueryType queryType, string queryString)
+        public RequestResult<IEnumerable<MessagingVerificationModel>> GetMessageVerifications(UserQueryType queryType, string queryString)
```

Comment:
The code in this method should use the same pattern as the one in the Admin project:

- initializing ResourcePayload to Enumerable.Empty
- populating ResourcePayload with dbResult.Payload.Select()